specification/storage/data-plane/Microsoft.BlobStorage/stable/2025-07-05/blob.json

```diff
@@ -0,0 +1,12474 @@
+                "x-ms-client-name": "AccountKind",
+                "type": "string",
+                "enum": [
+                  "Storage",
+                  "BlobStorage",
+                  "StorageV2",
+                  "FileStorage",
+                  "BlockBlobStorage"
+                ],
+                "x-ms-enum": {
+                  "name": "AccountKind",
+                  "modelAsString": false
+                },
+                "description": "Identifies the account kind"
+              },
+              "x-ms-is-hns-enabled": {
+                "x-ms-client-name": "IsHierarchicalNamespaceEnabled",
+                "type": "boolean",
+                "description": "Version 2019-07-07 and newer. Indicates if the account has a hierarchical namespace enabled."
+              }
+            }
+          },
+          "default": {
+            "description": "Failure",
+            "headers": {
+              "x-ms-error-code": {
+                "x-ms-client-name": "ErrorCode",
+                "type": "string"
+              }
+            },
+            "schema": {
+              "$ref": "#/definitions/StorageError"
+            }
+          }
+        }
+      },
+      "parameters": [
+        {
+          "$ref": "#/parameters/ContainerName"
+        },
+        {
+          "$ref": "#/parameters/Blob"
+        },
+        {
+          "name": "restype",
+          "description": "restype",
+          "in": "query",
+          "required": true,
+          "type": "string",
+          "enum": [
+            "account"
+          ]
+        },
+        {
+          "name": "comp",
+          "description": "comp",
+          "in": "query",
+          "required": true,
+          "type": "string",
+          "enum": [
+            "properties"
+          ]
+        }
+      ]
+    },
+    "/{containerName}/{blob}?comp=block": {
+      "put": {
+        "tags": [
+          "blockblob"
+        ],
+        "operationId": "BlockBlob_StageBlock",
+        "description": "The Stage Block operation creates a new block to be committed as part of a blob",
+        "consumes": [
+          "application/octet-stream"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/BlockId"
+          },
+          {
+            "$ref": "#/parameters/ContentLength"
+          },
+          {
+            "$ref": "#/parameters/ContentMD5"
+          },
+          {
+            "$ref": "#/parameters/ContentCrc64"
+          },
+          {
+            "$ref": "#/parameters/Body"
+          },
+          {
+            "$ref": "#/parameters/Timeout"
+          },
+          {
+            "$ref": "#/parameters/LeaseIdOptional"
+          },
+          {
+            "$ref": "#/parameters/EncryptionKey"
+          },
+          {
+            "$ref": "#/parameters/EncryptionKeySha256"
+          },
+          {
+            "$ref": "#/parameters/EncryptionAlgorithm"
+          },
+          {
+        
```

Comment:
This needs a better description.  It's such an unintuitive feature given how the rest of Azure works.

Reply:
Lol, I think github shit itself given the size of the the JSON file.  This was supposed to be added on the `description` for `FileRequestIntent` that just says `Valid value is backup` around line 11828.